test/test_p2p.py

```diff
@@ -10,9 +10,15 @@ def setUp(self):
 
         @bus.Object.interface('cockpit.Test')
         class TestObject(bus.Object):
+            _answer = 42
+
             @bus.Object.property('i')
             def answer(self):
-                return 42
+                return self._answer
+
+            @answer.setter
```

Comment:
I'm not a huge fan.  I feel like for 99% of our usecases here the value will have a somewhat static nature, and in that case, we could probably avoid the need to define the getter/setter functions like this.

As a first-go, this might be OK, though.

Reply:
I have copied this from Python's own `property()` decorator.  I am not sure I like it either.